composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "dusank/knapsack",
+  "name": "martinvenus/knapsack",
```

Comment:
@martinvenus Could we keep please the original author name? 

In my humble opinion, unless this package is declared as abandoned or archived, Information like this should be kept the same.

Reply:
I have lost hope that this will be merged someday and have made other changes to my repository. It's been a few years since I did this PR.

Reply:
I agree with you. But maybe this changed line is preventing the author to merge it, we do not know :)

In your fork main branch, you can change the name if desired, if a Pull Request is made against this repository which is considered still mainstream, I would advise otherwise. I guess at some point people will start using your fork if no reaction is noticed, like normally happened with many other open source projects that gets unattended in time.

Thanks for answering and regards!
David